### DISCUSSION TITLE
Octernships projects in a year

### DISCUSSION THREAD

**Discussion by PraveenRepswal** [community/community#73472] [GitHub Education]
### Hi everyone,

Hello everyone,
I want to know that for how much octernships I can apply in a year?
I have seen on some websites that I can apply for four octernships but I don't know that I can apply for for internships at a time or there is a limit of four internships for a whole year.
Can I apply for unlimited octernships in a year?

**Comment by Seol-JY** [comment]
You can submit up to four Octernship applications on a rolling basis throughout the year, as long as you remain a verified student.i.e., at any given time, you can have up to four ongoing opportunities applied.

I have got multiple opportunities. Can I accept all of them?
You can only accept one Octernship offer at a time, and all the other opportunities you applied for will be automatically rejected.

https://github.com/orgs/community/discussions/49380

**Reply by PraveenRepswal** [reply]
I didn't understand 'You can submit up to four Octernship applications on a rolling basis throughout the year, as long as you remain a verified student' clearly that's why I asked the above question.
**Above question:**
I want to know that for how much octernships I can apply in a year?
I have seen on some websites that I can apply for four octernships but I don't know that I can apply for for internships at a time or there is a limit of four internships for a whole year.
Can I apply for unlimited octernships in a year?

**Reply by Khushiyant** [reply]
Things to know about Octernships:
- Companies offer intership on single period of time (Let's call it a slot).
- In a single slot, you can apply to at most 4 companies
- There are many number of slots throughout the year
- Number of these slots is not fixed
